kodiak/pull_request.py

```diff
@@ -254,15 +254,19 @@ def __repr__(self) -> str:
                 )
             return MergeabilityResponse.NEEDS_UPDATE, self.event
 
-    async def update(self) -> None:
+    async def update(self) -> Optional[dict]:
         self.log.info("update")
         event = await self.get_event()
         if event is None:
             self.log.warning("problem")
-            return
-        await self.client.merge_branch(
+            return None
+        res = await self.client.merge_branch(
             head=event.pull_request.baseRefName, base=event.pull_request.headRefName
         )
+        if res.status_code > 300:
```

Comment:
feels like we should hook into the http client and have it automatically log non-ok errors

Reply:
Different PR I suppose